data_steward/analytics/cdr_ops/controlled_tier_qc/check_controlled_tier_part2.py

```diff
@@ -1471,10 +1478,12 @@ def query_template(table_era):
 result
 
 
+# # Results Summary
+
```

Comment:
Adding a markdown header for the summary so it shows up in the notebook table of contents.